CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - Update `CONTRIBUTING.md` with detailed installation and contribution instructions (#161)
 - Limit Python version to <3.12 in conda recipe (#161)
 - Change SessionDirectorySelectionDialog `.open()` call to `.exec()` (#163, #165)
+- Add `QDoubleSpinBox` support to `UISliderWidget` (#168)
```

Comment:
Please add a note that this breaks backward compatability - UISlider no longer accepts a QLabel

Reply:
Good point. Let's add this PR in the v2.0.0 release which is upcoming soon.

Reply:
Noted added to CHANGELOG.md

Reply:
After releasing v1.0.2 the changelog will need updating (merge master for this)